### PR TITLE
fix: add flag to check if variables file is archived

### PR DESCRIPTION
**Pull request by vLia**
## Pull request description 

Related to https://github.com/kubeshop/testkube/issues/3299

Adding flag to the test CRD to mark variables file as archived

The "running context" code was added by the generator

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: config/crd/bases/tests.testkube.io_tests.yaml [hunk 1]
```diff
@@ -584,6 +586,18 @@ spec:
                   preRunScript:
                     description: script to run before test execution
                     type: string
+                  runningContext:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_tests.yaml:589]: we don't use it in crd

File: config/crd/bases/tests.testkube.io_testsuites.yaml [hunk 2]
```diff
@@ -353,6 +353,18 @@ spec:
                     description: test kubernetes namespace (\"testkube\" when not
                       set)
                     type: string
+                  runningContext:
```
[the remark this answers is not in the document]
Comment (vLia) [reply config/crd/bases/tests.testkube.io_testsuites.yaml:356]: I will remove the references then

File: apis/testsuite/v2/zz_generated.deepcopy.go [hunk 3]
```diff
@@ -25,6 +25,21 @@ import (
 	runtime "k8s.io/apimachinery/pkg/runtime"
 )
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *RunningContext) DeepCopyInto(out *RunningContext) {
```
Comment (vsukhin) [inline apis/testsuite/v2/zz_generated.deepcopy.go:29]: we don't store it in crd

**Review by vsukhin — APPROVED** [review]
looks good, but we don't store Running Context in crd

**Pull request merged**